lib/engine/game/g_18_ga/game.rb

```diff
@@ -193,6 +203,47 @@ def event_close_companies!
         def remove_icon_from_waycross
           waycross_hex.tile.icons = []
         end
+
+        def georgia_railroad
+          corporation_by_id('GA')
+        end
+
+        def savannah
+          @savannah ||= hex_by_id('G13')
+        end
+
+        def charleston
+          @charleston ||= hex_by_id('E15')
+        end
+
+        def optional_hexes
+          return self.class::HEXES unless @optional_rules&.include?(:cotton_port)
+
+          new_hexes = {}
+          HEXES.keys.each do |color|
+            new_map = self.class::HEXES[color].transform_keys do |coords|
+              coords - COTTON_REMOVE_HEXES
+            end
+            COTTON_ADD_HEXES[color]&.each { |coords, tile_str| new_map[coords] = tile_str }
+            new_hexes[color] = new_map
+          end
+
+          new_hexes
+        end
+
+        def place_home_token(corporation)
+          return if corporation.tokens.first&.used == true
```

Comment:
Don't compare a boolean to true.
Do you need the safe navigation operator? The only way you could get a nil is if the corporation doesn't have any tokens.
```suggestion
          return if corporation.tokens.first.used
```

Reply:
yep, completely right. I think I copied this code from 1847AE. Thanks for that. Fixed.